core/src/main/scala/org/apache/spark/api/java/JavaDoubleRDD.scala

```diff
@@ -164,6 +166,20 @@ class JavaDoubleRDD(val srdd: RDD[scala.Double])
   def sum(): JDouble = srdd.sum()
 
   /**
+   * Returns the minimum element from this RDD as defined by
+   * the default comparator natural order.
+   * @return the minimum of the RDD
+   */
+  def min(): JDouble = super.min(com.google.common.collect.Ordering.natural())
```

Comment:
This doesn't need `super` does it? it isn't wrong but feels very slightly overspecific.